packages/sshnoports/lib/sshnp/sshnp.dart

```diff
@@ -137,6 +142,7 @@ abstract class SSHNP {
       required String localPort,
       String? remoteUsername,
       bool verbose = false,
+      required String localSshdPort,
```

Comment:
Same thing here `this.localSshdPort = 22`

Reply:
Not possible in a constructer

Reply:
used ' String localSshdPort = '22' instead